scripts/testBot/runAqaArgParse.py

```diff
@@ -1,22 +1,27 @@
 import argparse
 import json
 import sys
+import os
+
+dirList = next(os.walk('.'))[1]
```

Comment:
@2001asjad Did you try if this works? The file is under `scripts/testBot/` so I don't think this will work. Note the file is in TKG repo while the tests are actually in aqa-tests repo. So we may need to think about those issue.

Reply:
Yeah @sophia-guo I noticed it. Although the code was working fine in the TKG repo. I'll look into this and update you soon.